File: jadx-dump/sources/com/sparkappdesign/archimedes/mathexpression/enums/MEIssueType.java
package com.sparkappdesign.archimedes.mathexpression.enums;
/* loaded from: classes.dex */
public enum MEIssueType {
    Error,
    Warning,
    Info;

    @Override // java.lang.Enum, java.lang.Object
    public String toString() {
        switch (this) {
            case Error:
                return "Error";
            case Warning:
                return "Warning";
            case Info:
                return "Info";
            default:
                return "Unknown issue";
        }
    }
}

File: app/src/main/java/com/oddlyspaced/calci/mathexpression/enums/MEIssueType.kt
package com.oddlyspaced.calci.mathexpression.enums

enum class MEIssueType {
    Error,
    Warning,
    Info;

    override fun toString(): String {
        // @TODO See if this logic can be improved
        return when (this) {
            Error -> "Errpr"
            Warning -> "Warning"
            Info -> "Info"
            else -> "Unknown Issue"
        }
    }
}